src/components/Notifications/NotificationTypes/Events/EventStartedNotification.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import { i18n } from "./EventStartedNotification.i18n"
+import { EventStartedIcon } from "../../../Icon"
+import { NotificationItem } from "../../NotificationItem"
+import {
+  NotificationItemDescription,
+  NotificationItemTitle,
+} from "../../NotificationItem.styled"
+import {
+  CommonNotificationProps,
+  EventsStartedNotificationProps,
+} from "../../types"
+
+const EventsStartedNotification = React.memo(
+  (props: CommonNotificationProps<EventsStartedNotificationProps>) => {
+    const { notification, locale } = props
+    return (
+      <NotificationItem
+        image={{ image: <EventStartedIcon fontSize="inherit" /> }}
```

Comment:
It seems that all `EventStartedIcon` are using the property `fontSize` with `inherit`, what if we define it in `EventStartedIcon` as a default property?